### ISSUE TITLE
bug(windows): Help content does not appear for the "Keyman 18.0 Help"

### ISSUE THREAD

**Issue by dinakaranr** [keymanapp/keyman#12463]
### Describe the bug

Help content does not appear for the "Keyman 18.0 Help" topic

### Reproduce the bug

Steps to reproduce:

1. Installed the "Keyman-18.0.116.exe" file from the PR build.
2. Open the Keyman configuration app.
3. Navigate to the "Keyboard Layouts" tab. 
4. Install one or two keyboards (e.g. Khmer Angkor, Imperial Aramaic)
5. Navigate to the systems page. Click the "Install Keyboard or Dictionary"
6. Open the OSK and then select the keyboard.
7. Click the help icon on the OSK window.
8. The "Keyman Help" dialog appears and click the "Help Contents" link button.
Actual Results: The Keyman help dialog appears here, but no text content automatically appears for the "Keyman 18.0 Help" topic.
It seems to be a trivial issue
<img width="300" src="https://github.com/user-attachments/assets/d9d43353-f0ca-4dc3-ae98-f4c6c4677799">

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [X] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

18.0.116

### Operating system

Windows 10 OS

**Comment by rc-swag** [comment]
Verified same behaviour with Keyman 17.0

**Comment by rc-swag** [comment]
If you open the help by clicking on the keyman icon in the system tray and then selecting help from the pop up menu. Then click the "Help Contents" link, it opens without the error on the right side frame in the help window. First look I think opening the help windows via osk maybe loading the `FHelpJump`

FHelpJump

**Comment by mcdurdin** [comment]
'context_onscreenkeyboard' is an old reference; you can see the paths in docs/help, so for example I think this one should be `context/toolbox-onscreenkeyboard`, matching windows/docs/help/context/toolbox-onscreenkeyboard.md.

Looks like there may be a few others which are outdated as well? 

```
windows\src\engine\keyman\viskbd\UfrmVisualKeyboard.pas:
   641:       apKeyboard: TKeymanDesktopShell.OpenHelp('context_onscreenkeyboard');
   642:       apCharacterMap: TKeymanDesktopShell.OpenHelp('context_charactermap');
   643:       apFontHelper: TKeymanDesktopShell.OpenHelp('context_fonthelper');
   644:       apEntryHelper: TKeymanDesktopShell.OpenHelp('context_entryhelper');
  1549:   TKeymanDesktopShell.OpenHelpJump('context_onscreenkeyboard', frmKeyman7Main.ActiveKeyboard);
```

**Comment by rc-swag** [comment]
fixed in [#12646](https://github.com/keymanapp/keyman/pull/12646)
